### ISSUE TITLE
Installation documentation misleading/incorrect

### ISSUE THREAD

**Issue by 4lch4** [dynatrace-oss/dynatrace-gcp-function#97]
# Overview
Heyo, folks! As I mentioned in the other issue I created (#96), we recently tried installing the `dynatrace-gcp-function` to our GCP environment and we ran into an issue when providing the `setup.sh` script with our tenant URL.

## Details

According to the [official documentation][0] it states the following for the prerequisites:

![Installation-Prerequisites](https://user-images.githubusercontent.com/6931731/118200572-0a02bb80-b41b-11eb-8fce-fb61072fe918.png)

Then, when the script is running, it asks for the following values. For the tenant URL specifically, it mentions to refer to the prerequisites:

![image](https://user-images.githubusercontent.com/6931731/118200623-2b63a780-b41b-11eb-9b37-e51c563f6d17.png)

## Suspected Problem

The problem with that is the script doesn't allow for any of the recommended values to be true. The [regex on line 189][1] of `setup.sh` expects a URL to end with a slash and doesn't accept `/api` anywhere at the end. Unless I'm reading the regex incorrectly (which is entirely possible, in which case I apologize for making a fool of myself 😅), I'm reading it allows for:

- Up to 256 characters to define the DNS.
- Followed by an optional field for `/e/<36-digit-id>/`
- End string

## Wrap-Up

So, TL;DR: I believe the regular expression [on line 189][1] of the `setup.sh` script is either incorrect, or the prerequisites section of the install documentation needs to be updated to reflect the proper format for the tenant URI.

[0]: https://www.dynatrace.com/support/help/technology-support/cloud-platforms/google-cloud-platform/monitor-gcp-services-with-dynatrace/deploy-as-gcp-function/
[1]: https://github.com/dynatrace-oss/dynatrace-gcp-function/blob/master/scripts/setup.sh#L189

**Comment by dtwwolinski** [comment]
Hi @4lch4. Thank you for your finding. The documentation for "deploy-in-bash"  contains the `/api` suffix. The proper URL should be without `/api` suffix. The documentation will be updated.
